src/main/ruby/core/signal.rb

```diff
@@ -48,9 +48,10 @@ module Signal
     Names['CLD'] = Names['CHLD']
   end
 
-  # We want Numbers[SIGCHLD_VALUE] to refer to 'CHLD' not 'CLD',
-  # as CLD is not recognized by `new sun.misc.Signal("CLD")`
+  # We want Numbers[SIGCHLD_VALUE] to refer to CHLD and ABRT not to CLD and IOT,
+  # as CLD and IOT is not recognized by `new sun.misc.Signal(name)`
```

Comment:
*are* not

Reply:
Actually IOT is recognized on Linux but not macOS iirc.
It would be clearer to update the comment to say that
> Signal.signame(number) always returns the original and not the alias when they have the same signal numbers for CLD => CHLD and IOT => ABRT.